docs/user-manual/graphics/linear-workflow/textures.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Textures
+sidebar_position: 1
+---
+
+### sRGB Texture Handling  
+
+Textures that represent colors, such as Diffuse, Emissive, Specular, and Sheen, are typically stored in sRGB space to maintain color accuracy and reduce banding. When used by the engine, these textures are automatically converted from sRGB to linear space for correct lighting calculations. This conversion is performed by the GPU efficiently at no extra cost, provided that the texture is created using an sRGB format.  
+
+#### **Specifying sRGB Encoding for Textures**  
+
+When loading a texture asset that represents colors in sRGB space, it is important to specify sRGB encoding. The following example demonstrates how to create an asset with sRGB encoding:  
+
+```javascript
+new pc.Asset(
+    'color',
+    'texture',
+    { url: 'heart.png' },
+    { encoding: 'srgb' }
+);
+```
+
+#### **Marking sRGB Textures in the Editor**  
+
+When working in the Editor, ensure that the texture is marked as **sRGB** in the inspector panel. This guarantees that the engine correctly interprets the texture as sRGB and applies the necessary conversion to linear space.
```

Comment:
"ensure that color texture is marked" - probably worth mentoning

Reply:
done thanks